src/open_inwoner/react/modules/KVKBranchSelectorModule/KVKBranchSelectorModule.stories.tsx

```diff
@@ -0,0 +1,186 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { IntlProvider } from 'react-intl'
+import { KVKBranchSelector } from '@react/components/KVKBranchSelector/KVKBranchSelector'
+
+const meta: Meta<typeof KVKBranchSelector> = {
+  title: 'React/KVKBranchSelectorModule',
```

Comment:
I think that it would be nice to have a separation between modules and components.


```suggestion
  title: 'React/Modules/KVKBranchSelectorModule',
```


This can than be added to the component.
`title: 'React/Component/KVKBranchSelector,`

Reply:
Not if we do not want to add Module stories anyway, so what to do? The module stories are really only meant for developers. And in this case the BranchSelector is not an NLDS component, so it could actually be good to make a story for it, in order to give the component 'back to the NLDS community'